src/main/java/org/spongepowered/api/world/World.java

```diff
@@ -157,22 +157,12 @@
     Dimension getDimension();
 
     /**
-     * Gets the random seed for this world.
-     *
-     * @return The seed
-     */
-    long getWorldSeed();
```

Comment:
Wouldn't it be best to keep world seed here namely for the fact that you could override the seed by World if you want?